src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/haflows/HaFlowPingSpec.groovy

```diff
@@ -129,7 +127,6 @@ class HaFlowPingSpec extends HealthCheckSpecification {
 
         cleanup:
         haFlow && haFlow.delete()
-        islHelper.restoreIsl(islToFail)
         database.resetCosts(topology.isls)
```

Comment:
We can omit database.resetCosts(topology.isls) as it is executed in the scope of cleanup action for broken ISLs, don't we?